### DISCUSSION TITLE
Modify nvim-tree settings

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [LazyVim/LazyVim#4558] [Q&A] [answered]
Hi,
I started using LazyVim and I still parsing the whats and hows.

I want to change nvim-tree settings as it hides majority of the things in a directory. I followed the "Configuration" advice and added a "nvim-tree.lua" file under the plugin directory, see it below. But, something is not ok. When I save the file I see the message about the the config has changed, but no visible sign of it. The width is the same and hidden files are still hidden. I want to see everything in nvim-tree.

What do I wrong here?

```
return {
  {
    "nvim-tree/nvim-tree.lua",
    opts = {
      git = {
        enable = true,
        ignore = false,
      },
      view = {
        width = 100,
        side = "left",
      },
    },
  },
}
```
<img width="513" alt="Screenshot 2024-10-09 at 08 17 16" src="https://github.com/user-attachments/assets/deff2452-8e3d-46a1-a748-835399fb9654">

**Comment by dpetka2001** [comment, marked as answer]
LazyVim use `neo-tree`, not `nvim-tree`. Check the `neo-tree` docs for the name of the setting that you want.

**Reply by Andras-Csanyi** [reply]
Well, this is a massive 🤦 at my side. :D Thanks, for correcting me. I really appreciate it.
